lib/SPIRV/OCLToSPIRV.cpp

```diff
@@ -1890,6 +1897,37 @@ void OCLToSPIRVBase::visitSubgroupAVCBuiltinCallWithSampler(
       &Attrs);
 }
 
+void OCLToSPIRVBase::visitCallSplitBarrierINTEL(CallInst *CI,
+                                                StringRef DemangledName) {
+  auto Lit = getBarrierLiterals(CI);
```

Comment:
nit:
Some static analyzers will be paranoid about nullptrness of CI. Even there is an appropriate check in the call stack before  `visitCallSplitBarrierINTEL` can we also put an assertion here?

Reply:
I'm willing to make a change here but I'm not sure what to type.  This particular split barrier addition is purposefully very similar to `visitCallBarrier`, which has no assertion.  Is this suggestion something that would be more appropriate for a separate PR?

Reply:
Okay, there is actually no need for the assertion, since there is the appropriate check earlier in the stack.